apps/frontend/src/app/xformation-editor/xformation-editor.component.html

```diff
@@ -1,13 +1,17 @@
-<form [formGroup]="profileForm">
+<form [formGroup]="transform">
+
   <label for="name">Name: </label><br>
-  <input id="name" type="text" formControlName="name" />
-  <br><br>
-  <label for="description">Description: </label><br>
-  <input id="description" type="textarea" formControlName="description" size="40" />
-  <form [formGroup]="profileForm" (ngSubmit)="onSubmit()"></form><br>
+  <input id="name" type="text" formControlName="name" required="true" size="40"><br><br>
```

Comment:
Better move the validators to the reactive form side in the ts file.

Just add this to the FormGroup
```ts
name: new FormControl('', [Validators.required, Validators.maxLength(40)])
```